src/interp.cc

```diff
@@ -1214,6 +1214,23 @@ ValueTypeRep<T> Xchg(ValueTypeRep<T> lhs_rep, ValueTypeRep<T> rhs_rep) {
   return rhs_rep;
 }
 
+// i(8,16,32,64) f(32,64) X (2,4,8,16) splat ==> v128
+template <typename T, typename V>
+ValueTypeRep<T> SimdSplat(V Lane_Data) {
```

Comment:
should be all lowercase, like `lane_data`, but we can clean that up in a future PR.

Reply:
@binji 
Got it, thanks!